src/main/java/org/cyclops/energeticsheep/entity/EntityEnergeticSheepConfig.java

```diff
@@ -107,7 +115,7 @@ public int getForegroundEggColor() {
     public void onRegistered() {
         super.onRegistered();
         for (Biome biome : Biome.REGISTRY) {
-            EntityRegistry.addSpawn(EntityEnergeticSheep.class, 5, 2, 4, EnumCreatureType.CREATURE, biome);
+            EntityRegistry.addSpawn(EntityEnergeticSheep.class, spawnChance, 2, 4, EnumCreatureType.CREATURE, biome);
```

Comment:
This is actually not a chance, but a spawn weight (values have no upper limit). So I would rename this option (and description accordingly).

Reply:
I am finding it rather difficult to concisely describe what this option does, without explaining spawning mechanics.